board/at91sam9m10g45ek/at91sam9m10g45ek.c

```diff
@@ -132,34 +132,86 @@ static void recovery_buttons_hw_init(void)
 }
 #endif /* #if defined(CONFIG_NANDFLASH_RECOVERY) || defined(CONFIG_DATAFLASH_RECOVERY) */
 
-static int ek_special_hw_init(void)
+#ifdef CONFIG_USER_HW_INIT
+//*----------------------------------------------------------------------------
+//* \fn    nedap_usb_fix
+//* \brief This function fixes an issue with the usb
+//*----------------------------------------------------------------------------*/
```

Comment:
No clue. Copied from oe-classic. It is not used anymore (almost not...). Will remove.